doozer/doozerlib/olm/bundle.py

```diff
@@ -581,3 +591,15 @@ def target(self):
     @property
     def valid_subscription_label(self):
         return self.operator_csv_config['valid-subscription-label']
+
+    def delivery_labels_match(self, bundle_image_pullspec):
+        cmd = f"oc image info {bundle_image_pullspec} -o json"
+        rc, out, err = exectools.cmd_gather(cmd)
+        if rc != 0:
+            raise ValueError(f"Error running {cmd}: {err}")
+        image_info = json.loads(out)
+        image_labels = image_info['config']['config']['Labels']
+        for label, value in self.redhat_delivery_tags.items():
+            if image_labels.get(label, '') != value:
+                return False
+        return True
```

Comment:
fine, but clearer to me:
```suggestion
        image_labels = json.loads(out)['config']['config']['Labels']
        return all(image_labels.get(label) == val for label, val in self.redhat_delivery_tags.items())
```